src/schemas.ts

```diff
@@ -100,4 +100,14 @@ const removeMessage = {
   },
 };
 
-export { getChat, publishMessage, patchMessage, removeMessage };
+/**
+ * JSON schema on DELETE clear chat route for request and response
+ */
+const clearChat = {
+  params: { $ref: 'http://graasp.org/chat/#/definitions/itemIdParam' },
+  response: {
+    200: { $ref: 'http://graasp.org/chat/#/definitions/chat' },
```

Comment:
Maybe you'll need to change this depending on your returned value.

Reply:
@pyphilia I now return something like:
```json
{
    "id": "39370f67-2153-4ab9-9679-b1966542d27d",
    "messages": []
}
```

So it is still compliant with the schema ? This is what is returned for an empty chat on a `GET` request.